src/content/drafts/eslint-html-plugin.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: html-eslint now supports ESLint language plugin
+teaser: "Bringing HTML linting to ESLint’s new language-agnostic system."
+tags:
+  - Open Source
+  - Linting
+  - Parsing
+authors:
+  - yeonjuan
+categories:
+  - Announcements
+---
+
+In 2024, [ESLint announced](/blog/2024/07/whats-coming-next-for-eslint/) its plan to become a language-agnostic linter that is capable of linting languages other than JavaScript.
+Since then, we've seen official support added for JSON, Markdown, and most recently, CSS.
```

Comment:
```suggestion
Since then, we've seen official support added for [JSON and Markdown](https://eslint.org/blog/2024/10/eslint-json-markdown-support/), and most recently, [CSS](https://eslint.org/blog/2025/02/eslint-css-support/).
```